internal/pkg/version/check_remote.go

```diff
@@ -43,8 +38,8 @@ func (c *checker) CheckIfLatest(currentVersion string) error {
 	}
 
 	// Disabled by ENV
-	if value, _ := c.envs.Lookup(EnvVersionCheck); strings.ToLower(value) == "false" {
-		return errors.New(fmt.Sprintf(`skipped, disabled by ENV "%s"`, EnvVersionCheck))
+	if c.skip {
+		return errors.New(`skipped, check disabled`)
 	}
```

Comment:
Tu bolo posledne miesto kde bola pouzita `.Envs()` dependency.
Vsetko ostatne je uz prerobene na konfiguracne struktury a automaticke bindovanie flags/envs.

Nahradil som to za flag `--check-version`.

Kazdy CLI flag sa da zadat ako `KBC_....`, teda pre uzivatela sa nic nemeni, nastavi to rovnako cez ENV `KBC_VERSION_CHECK` (alebo po novom aj cez flag).